arm-machinelearning/2017-01-01/swagger/webservices.json

```diff
@@ -0,0 +1,1105 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The response payload is identical to the response payload that is returned by the GET operation. The response includes the Provisioning State and the Azure-AsyncOperation header. To get the progress of the operation, call GET operation on the URL in Azure-AsyncOperation header field. For more information about Asynchronous Operations, see https://msdn.microsoft.com/en-us/library/mt742920.aspx.",
+            "schema": {
+              "$ref": "#/definitions/WebService"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "WebServices"
+        ],
+        "operationId": "WebServices_Remove",
+        "description": "Deletes the specified web service.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WebServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },        
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. Note that a 202 status is returned even if the service did not exist."
+          },
+          "204": {
+            "description": "No Content."
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearning/webServices/{webServiceName}/CreateRegionalBlob":{
+      "post": {
+        "tags": [
+          "WebServices"
+        ],
+        "operationId": "WebServices_CreateRegionalWebServiceProperties",
+        "description": "Creates an encrypted credentials parameter blob for the specified region. To get the web service from a region other than the region in which it has been created, you must first call Create Regional Web Services Properties to create a copy of the encrypted credential parameter blob in that region. You only need to do this before the first time that you get the web service in the new region.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WebServiceNameParameter"
+          },
+          {
+            "in": "query",
+            "name": "region",
+            "type": "string",
+            "description": "The region for which encrypted credential parameters are created.",
+            "required": true
+          },        
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```

Comment:
@NonStatic2014 yes, it should return 202. But, because it's **long-running operation** it should also specify a **terminal** status code. Which is one of these 
- **200 OK** Standard response for successful HTTP requests. The actual response will depend on the request method used. In a GET request, the response will contain an entity corresponding to the requested resource. In a POST request, the response will contain an entity describing or containing the result of the action.
- **201 Created** The request has been fulfilled, resulting in the creation of a new resource.
- **204 No Content** The server successfully processed the request and is not returning any content.

**202** is not a terminal status code: The request has been accepted for processing, but the processing has not been completed. The request might or might not be eventually acted upon, and may be disallowed when processing occurs.

See this documentation https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md, search for LongRunning.